common/reviews/api/tools.api.md

```diff
@@ -2728,6 +2728,8 @@ interface PlanarFreehandROIAnnotation extends Annotation {
         polyline: Types_2.Point3[];
         label?: string;
         isOpenContour?: boolean;
+        isOpenCardiacAnnotation?: boolean;
+        openCardiacAnnotationVectorToPeak?: Types_2.Point3[];

```

Comment:
Are you claiming that the horseshoe shape is only for cardiac and not being used anywhere else? I think it makes sense to say something like more general, maybe is horseShoe shaped? that is aweful too but still not claiming that we know it all and this is being used only in cardiac